### ISSUE TITLE
handling gaps in timeseries in graphs

### ISSUE THREAD

**Issue by kaspernissen** [humio/humio2grafana#4]
Grafana version: 2.6.2

Humio query:
```
log_type=reqresp AND k8s.labels.squad=nasa
| eval(grpc.responseTime_ns=grpc.responseTime_ns/1000000) | timechart(k8s.labels.app,function=percentile(field=grpc.responseTime_ns, percentiles=[95], as=grpc))
```

Screenshot of resulting graph in grafana:
![screen shot 2018-01-10 at 13 31 23](https://user-images.githubusercontent.com/4429108/34773091-b81e3014-f60a-11e7-978b-21292970b969.png)

Same in Humio:
![screen shot 2018-01-10 at 13 31 14](https://user-images.githubusercontent.com/4429108/34773093-b9d9e736-f60a-11e7-94d4-fbbfcbb04062.png)

Grafana can be configured to deal with null values under the "Display" tab. It don't seem that the current implementation support this functionality

![screen shot 2018-01-10 at 13 33 22](https://user-images.githubusercontent.com/4429108/34773154-f829fd78-f60a-11e7-9a61-d546da7b5d0a.png)

**Comment by pmech** [comment]
hi. I actually think that Grafana's default makes more sense than ours. So I tend to see this as a feature...

I'm unsure about what you would want us do change in our plugin regarding this.

**Comment by kaspernissen** [comment]
Not sure, what grafana expects when `null as zero` is configured. Maybe it needs something more in order to be able to work?

**Comment by pmech** [comment]
This is actually something that we need to address in Humio I think, and not so easy to fix short term.

I this blocking you?

**Comment by kaspernissen** [comment]
No, it's not blocking me. The data comes in, but the graphs are not that pretty. Other issues have higher priority from my perspective.

**Comment by nikobearrr** [comment]
my 2¢: I would prefer to see it as on the grafana picture instead of the humio. I see that it's not as "pretty" to have gaps, but definitely better to see that there is no info compared to just draw lines which can mask the missing data. 

In case that this is a thing the team will look into, please provide the option to switch between the two
